### PR TITLE
Simple scroll posts

### PR DESCRIPTION
**Pull request by motuncoded**
I created a post feed UI using the nextjs and typescript  environment to render the post feeds with the integration of JSON Placeholder API posts

### CODE REVIEW

File: pages/api/hello.ts
Comment (Asylcreek) [inline pages/api/hello.ts:1]: Should this file be here?

File: pages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+export default function Home() {
+  type Post = {
+  id: number;
+  title: string;
+  body: string;
+  userId: number;
+};
+
+    //API 
+    const BASE_URL = "https://jsonplaceholder.typicode.com/posts";
```
Comment (Asylcreek) [inline pages/index.tsx]: Move this outside of the Home function

File: pages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+
+
+
+  const fetchPosts = async() => {
+    setLoading(true)
+    const response = await fetch(BASE_URL);
+    const posts = await response.json();
+    setPosts(posts)
+    setLoading(false)
```
Comment (Asylcreek) [inline pages/index.tsx]: What happens when there's an error in fetching the posts?

File: styles/globals.css [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+    display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 1rem 0;
+
+}
+.postWrapper{
+    background-color: var(--dark);
+  width: min(700px, (100% - 2rem));
```
Comment (Asylcreek) [inline styles/globals.css]: Can you explain what's happening here? Wouldn't you need the `calc` function to achieve the arithmetic here?

**Review by Asylcreek — CHANGES_REQUESTED** [review]
Please integrate [Prettier](https://prettier.io/) into your app

File: styles/globals.css (hunk 3 above)
Comment (motuncoded) [reply styles/globals.css]: Will resolve

File: pages/api/hello.ts
Comment (motuncoded) [reply pages/api/hello.ts:1]: ok

**Review by Asylcreek — APPROVED** [review]

**Comment by motuncoded** [comment]
Thank you for the approval

**Pull request merged**